test/python/circuit/test_parameters.py

```diff
@@ -1466,6 +1483,263 @@ def test_parameter_expression_grad(self):
             self.assertEqual(expr.gradient(x), 2 * x)
             self.assertEqual(expr.gradient(x).gradient(x), 2)
 
+    def test_floor(self):
+        """Test ParameterExpression floor."""
+        a = 1.3
+        b = 2
+        x = Parameter('x')
+        y = Parameter('y')
+        expr = numpy.floor(x*y)
+        bexpr = expr.bind({x: a, y: b})
+        self.assertEqual(float(bexpr), numpy.floor(a * b))
+
+    def test_ceil(self):
+        """Test ParameterExpression ceil."""
+        a = 1.3
+        b = 2
+        x = Parameter('x')
+        y = Parameter('y')
+        expr = numpy.ceil(x*y)
+        bexpr = expr.bind({x: a, y: b})
+        self.assertEqual(float(bexpr), numpy.ceil(a * b))
```

Comment:
Can you add a test that these functions will raise if the expression is bound with a complex value?

Reply:
@kdk Should these raise if the underlying sympy method doesn't raise?

Reply:
I think it depends on the case. Min/Max should. For ceil/floor, I would've expected a TypeError but it looks like sympy will take the ceil/floor of the real and imaginary parts which is okay.  I'm good either way, so long the behavior over complex values is covered in the tests and docstring.

Reply:
I added cases for complex values